crates/bevy_ecs/src/query/iter.rs

```diff
@@ -216,12 +216,24 @@ where
 
 /// An iterator over `K`-sized combinations of query items without repetition.
 ///
+/// A combination describes the number of possible arrangements of a collection of
+/// items where order does not matter.
+///
+/// The size of the items array output by each iteration of the loop is parameterized by `K`.
+/// The number of total items input to the function is defined as `N`.
+///
+/// For example, given the list [1, 2, 3, 4], where `K` is 2, the combinations without repeats are
+/// [1, 2], [1, 3], [1, 4], [2, 3], [2, 4], [3, 4].
+/// And in this case, `N` would be defined as 4 since the size of the input list is 4.
+///
 /// In this context, a combination is an unordered subset of `K` elements.
```

Comment:
This comment should be better integrated into the added docs. It's a bit mathy, but useful as a synonym.

Reply:
I can try to find a way to make it all flow together!